gradio/routes.py

```diff
@@ -875,8 +673,6 @@ def read_main():
     @app.on_event("startup")
     async def start_queue():
         if gradio_app.get_blocks().enable_queue:
-            if gradio_api_url:
-                gradio_app.get_blocks()._queue.set_url(gradio_api_url)
```

Comment:
Now the `gradio_api_url` argument of `mount_gradio_app` is no longer used,
but I kept it for now because its deletion makes the CI fail, probably at building the document.
What's the best way?

```python
def mount_gradio_app(
    app: fastapi.FastAPI,
    blocks: gradio.Blocks,
    path: str,
    gradio_api_url: str | None = None,
    app_kwargs: dict[str, Any] | None = None,
)
```

Reply:
Got it! -> https://github.com/gradio-app/gradio/pull/5165/commits/29a74c6634bcf93dc11a1261f06daa17b16afd40